content/en/docs/armory-agent/armory-agent-install/install-agent-service-helm/index.md

```diff
@@ -0,0 +1,250 @@
+{{< /tabs >}}
+
+### Confirm success
+
+Create a pipeline with a **Deploy manifest** stage. You should see your target cluster available in the **Accounts** list. Deploy a static manifest.
+
+## Set proxy settings
+
+The `env` parameters are optional and only need to be used if Armory Enterprise is behind an HTTP(S) proxy. If you need to set more than one of the `env` parameters, you must increment the index value for the parameters. For example: `env[0].name="HTTP_PROXY`, `env[1].name="HTTPS_PROXY"`, and `env[2].name="NO_PROXY"`.
+
+{{< prism lang="bash" line="4-6">}}
+helm install armory-agent armory-charts/agent-k8s-full \
+--create-namespace \
+--namespace=<agent-namespace> \
+--set env[0].name=”HTTP_PROXY”,env[0].value="<hostname>:<port>" \
+,env[1].name=”HTTPS_PROXY”,env[1].value="<hostname>:<port>" \
+,env[2].name=”NO_PROXY”,env[2].value="localhost,127.0.0.1,*.spinnaker"
+{{< /prism >}}
+
+Alternatively, you can create a `values.yaml` file to include the parameters:
+
+{{< prism lang="yaml" >}}
+env:
+  - name: HTTP_PROXY
+    value: <hostname>:<port>
+  - name: HTTPS_PROXY
+    value: <hostname>:<port>
+  - name: NO_PROXY
+    value: localhost,127.0.0.1,*.spinnaker
+{{< /prism >}}
+
+With the file, you can avoid setting individual `env` parameters in the `helm install` command. Instead include the `--values` parameter as part of the Helm install command:
+
+```bash
+--values=<path>/values.yaml
+```
+
+## Set a custom Agent image registry
+
+If you need to store the Armory Agent image in a different registry, you can specifiy that repository using the following settings:
+
+{{< prism lang="bash">}}
+--set image.respository=<repo-name>,image.imagePullPolicy=IfNotPresent, /
+image.imagePullSecrets=<secret>
+{{< /prism>}}
+
+Alternatively, you can create a `values.yaml` file to include the settings:
+
+{{< prism lang="yaml" >}}
+image:
+  repository: <repo-name>
+  imagePullPolicy: IfNotPresent
+  imagePullSecrets: <secret>  
+{{< /prism >}}
+
+Include the `--values` parameter as part of the Helm install command:
+
+{{< prism lang="bash" >}}
+--values=<path>/values.yaml
+{{< /prism >}}
+
+## Custom configuration
+
+If you need greater flexibility, you can override any supported [configuration option]({{< ref "agent-options#configuration-options" >}}), either by setting values using the command line or by setting values in an `armory-agent.yml` file.
+
+### Set config values using the command line
+
+You can update configuration options via the command line by using `--set config.<option>`. For example, if you want to set `logging.level` to `debug`, use `--set config.logging.level=debug` to do that.
+
+
+### Set configuration in a file
+
+You can also override default settings in your own `armory-agent.yml` file. For example, if you want to modify the default logging settings, create an `armory-agent.yml` file with the following content:
+
+{{< prism lang="yaml" >}}
+config:
```

Comment:
when using agentyml the format is the same as the kubectl install method, there is no config prefix.